src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  type Control,
+  Controller,
+  type UseFormResetField,
+  type UseFormWatch,
+} from 'react-hook-form'
+
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { Button } from 'lib/ui/Button.js'
+import type { ClimateSettingScheduleFormFields } from 'lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.js'
+import { FormField } from 'lib/ui/FormField.js'
+import { InputLabel } from 'lib/ui/InputLabel.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { ClimateModeMenu } from 'lib/ui/thermostat/ClimateModeMenu.js'
+import { TemperatureControlGroup } from 'lib/ui/thermostat/TemperatureControlGroup.js'
+
+interface ClimateSettingScheduleFormClimateSettingProps {
+  title: string
+  control: Control<ClimateSettingScheduleFormFields>
+  watch: UseFormWatch<ClimateSettingScheduleFormFields>
+  resetField: UseFormResetField<ClimateSettingScheduleFormFields>
+  deviceId: string
+  onBack: () => void
+  onCancel: (() => void) | undefined
+  onSave: () => void
+}
+
+export function ClimateSettingScheduleFormClimateSetting({
+  title,
+  control,
+  watch,
+  resetField,
+  deviceId,
+  onBack,
+  onCancel,
+  onSave,
+}: ClimateSettingScheduleFormClimateSettingProps): JSX.Element {
+  const { device } = useDevice({
+    device_id: deviceId,
+  })
+
+  const hvacModeSetting = watch('hvacModeSetting')
+
+  return (
+    <>
+      <ContentHeader
+        title={title}
+        onBack={onBack}
+        subheading={device?.properties.name}
+      />
+      <div className='seam-main'>
+        <div className='seam-climate-setting-schedule-form-climate-setting'>
+          <div className='seam-content'>
+            <div>
+              <InputLabel>{t.climateSetting}</InputLabel>
+              <span className='seam-label'>{t.climateSettingSubHeading}</span>
+            </div>
+            <FormField>
+              <Controller
+                name='hvacModeSetting'
+                control={control}
+                render={({ field: { value, onChange } }) => {
+                  return (
+                    <ClimateModeMenu
+                      mode={value}
+                      onChange={(mode) => {
+                        resetField('setPoints')
+                        onChange(mode)
+                      }}
+                    />
+                  )
+                }}
+              />
+            </FormField>
+
+            {hvacModeSetting !== 'off' && (
+              <FormField className='seam-climate-setting-slider-container'>
```

Comment:
This nesting is getting a little deep to grok. Consider making the stuff inside `<div className='seam-content'>` its own component (inside the same file)